package.json

```diff
@@ -6,7 +6,7 @@
   "main": "run.js",
   "scripts": {
     "start": "node run.js",
-    "dev": "PERMISSIVE_CORS=1 node-dev run.js",
+    "dev": "PERMISSIVE_CORS=1 nodemon run.js",
```

Comment:
any reason for switching these?

Reply:
yes, I wrote: 
> replace node-dev with nodemon because on my computer node-dev stopped reloading on changes for unknown reason
furthermore, nodemon is regularly updated (3 month ago) and has 5,261,389 downloads weekly, whereas node-dev has only 54,353 and was last updated 8 month ago :)